runner/src/server/plugins/applicationStatus/index.ts

```diff
@@ -30,14 +31,19 @@ const index = {
           ],
           handler: async (request: HapiRequest, h: HapiResponseToolkit) => {
             const { statusService, cacheService } = request.services([]);
+            let queueService;
+            if (config.enableQueueService) {
+              const { queueStatusService } = request.services([]);
+              queueService = queueStatusService;
```

Comment:
`runner/src/server/index.ts` - take a look at server.registerService. this is where you should add queueService